src/main/java/com/example/Library/mapper/BookStatsMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.example.Library.mapper;
+
```

Comment:
Imo do BookStats nie potrzeba mappera, są to wartości wyliczone typowo pod endpoint, ale może się mylę